docs/src/components/ComponentDoc/ComponentExample/ComponentExample.tsx

```diff
@@ -31,14 +31,10 @@ interface ComponentExampleState {
   knobs: Object
   themeName: string
   componentVariables: Object
-  exampleElement?: JSX.Element
-  handleMouseLeave?: () => void
-  handleMouseMove?: () => void
+  handleMouseLeave: () => void
+  handleMouseMove: () => void
   sourceCode: string
-  markup: string
-  error?: string
   showCode: boolean
-  showHTML: boolean
```

Comment:
It's very confusing, this variable wasn't used at all

Reply:
I believe this is some left over, as we are showing the HTML alongside with the code editing... My proposal is to just remove it..

Reply:
Yes, I have already done this 😁